web/controllers/consent_controller.ex

```diff
@@ -3,6 +3,7 @@ defmodule ResearchResource.ConsentController do
 
   alias ResearchResource.Redcap.RedcapHelpers
   alias ResearchResource.Qualtrics.QualtricsHelpers
+  alias ResearchResource.User
```

Comment:
`alias ResearchResource.{Redcap.RedcapHelpers, Qualitrics.QualtricsHelpers, User}`
I prefer this shorter syntax
Not a big deal though